docs/pages/setup/reference/backends.mdx

```diff
@@ -423,7 +423,7 @@ teleport:
 
     # Teleport by default stores stores audit events with an AWS TTL of 1 year.
     # This value can be configured as shown below. If set to 0 seconds, TTL is disabled.
-    audit_retention_period: 1y
+    audit_retention_period: 365d
```

Comment:
Can you also add this configuration attribute to the Config Reference here - https://goteleport.com/docs/setup/reference/config/